data/json/npcs/robofac/robofac_intercom/robofac_intercom_missions.json

```
@@ -507,7 +507,8 @@
   {
     "id": "MISSION_ROBOFAC_INTERCOM_3_ADVICE_Location",
     "type": "talk_topic",
-    "dynamic_line": "I don't believe I'm cleared to-\"  You hear brief chatter.  \"Apologies; I was mistaken.  The manufacturer was, technically speaking, kept secret from the public.  As a result, it operated using an office tower as a front, and was located underground.  An elevator in the back of the building led to the basement, but it was likely destroyed in the damage - as a result, you will need to find a way down yourself.  While there were stairs as well, it's highly likely that they have been destroyed.\"\n\n\"We advise bringing a way to safely enter and exit the lower level; a grappling hook would work well for this purpose.  Given the damage, the elevator shaft itself probably remains intact, and would be your likeliest point of entry to the underground portion of the building."
+    "//": "Restore previous dialogue if we ever get multi-story rappelling",
```

```suggestion
    "//": "Restore previous dialogue if we ever get multi-story rappelling",
```

Multistory rappelling likely won't help here without other changes since the elevator shafts are now a special cased terrain that cannot be descended without it trapping you and teleporting you to somewhere else in LIXA ground floor.